src/Plugin/QueueWorker/EntitySyncTo.php

```diff
@@ -72,7 +85,12 @@ public function processItem($data) {
       ->getStorage($data['entity_type'])
       ->load($data['entity_id']);
 
-    $this->syncTo->sync($entity);
+    $remoted_id = $this->syncTo->sync($entity);
+
+    // Now dispatch an event to allow other modules to determine what to do with
+    // the result of the sync.
+    $event = new PostSyncEvent($entity, $remoted_id);
```

Comment:
**CHANGE REQUEST**

Can we rename this from `remoted_id` to `remote_id`?

Reply:
Ooops, yeah.